dspace_import.py

```diff
@@ -96,15 +97,16 @@ def read_metadata():
     """
     global metadatavalue
     metadatavalue_json = read_json('metadatavalue.json')
-    if metadatavalue_json:
-        for i in metadatavalue_json:
-            key = (i['resource_type_id'], i['resource_id'])
-            # replace separator @@ by ;
-            i['text_value'] = i['text_value'].replace("@@", ";")
-            if key in metadatavalue.keys():
-                metadatavalue[key].append(i)
-            else:
-                metadatavalue[key] = [i]
+    if not metadatavalue_json:
+        return
```

Comment:
Maybe it will be better if the user get some notice that we are not going to import metadatavalue, because json is null.